legate/core/runtime.py

```diff
@@ -131,12 +131,18 @@ def update(self, **kwargs: Any) -> None:
     def remove(self, key: str) -> None:
         del self._entries[key]
 
-    def __repr__(self) -> str:
+    def machine_string(self) -> str:
```

Comment:
machine_string and user_string have roughly the same information. Why separate methods?